src/components/task-item/template.html

```diff
@@ -0,0 +1,34 @@
+<link
+  rel="stylesheet"
+  href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0/css/all.min.css"
```

Comment:
Professor mentioned having some kind of handling in case a link is no longer accessible online, this might be a good place to implement something like that (have a handler or local reference?)

Reply:
good point: backlog